docs/File-Formats.md

```diff
@@ -605,7 +605,7 @@ A more detailed example MAF can be found in [study_es_0](https://raw.githubuserc
 23. **Match_Norm_Validation_Allele2 (Optional)**: Secondary data from orthogonal technology.
 24. **Verification_Status (Optional)**: Second pass results from independent attempt using same methods as primary data source.
 25. **Validation_Status (Optional)**:  -- "Valid" or "Unknown".
-26. **Mutation_Status (Optional)**: Ideally "Somatic".
+26. **Mutation_Status (Optional)**: Ideally "Somatic", but "Germline" is also possible and will be displayed in mutations tab. Values "None", "LOH" and "Wildtype" will not be loaded in cBioPortal.
```

Comment:
Apologies for new comment, but can we just make this: 
"Somatic" or "Germline" are supported by the UI in Mutations tab. "None", "LOH" and "Wildtype" will not be loaded. Other values will be displayed as text.

You can also add this change/commit to the other PR (https://github.com/cBioPortal/cbioportal/pull/3677)